app/forms/journeys/further_education_payments/teaching_hours_per_week_form.rb

```diff
@@ -0,0 +1,44 @@
+module Journeys
+  module FurtherEducationPayments
+    class TeachingHoursPerWeekForm < Form
+      attribute :teaching_hours_per_week, :string
+
+      validates :teaching_hours_per_week,
+        inclusion: {in: ->(form) { form.radio_options.map(&:id) }, message: i18n_error_message(:inclusion)}
+
+      def question
+        "On average, how many hours per week are you timetabled to teach at #{college.name} during the current term?"
```

Comment:
Not a blocker, but to note, we should be using I18n for this type of content strings, as we do with error messages.

Reply:
yep I will go back and change these